builder/builder-next/executor_linux.go

```diff
@@ -119,6 +119,11 @@ func (iface *lnInterface) init(c *libnetwork.Controller, n *libnetwork.Network)
 	iface.ep = ep
 }
 
+// TODO(neersighted): Unstub Sample(), and collect data from the libnetwork Endpoint.
+func (iface *lnInterface) Sample() (*network.Sample, error) {
```

Comment:
> Can you move this commit "before" the buildkit update?

So that this commit can be squashed with the buildkit update (so that the commit compiles)